specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/examples/capacityReservationExamples/CapacityReservationGroup_Get.json

```diff
@@ -23,6 +23,7 @@
               {
                 "name": "myCapacityReservation1",
                 "utilizationInfo": {
+                  "currentCapacity": 5,
```

Comment:
In this and other example files for capacity reservation change expand parameter wherever specified to $expand.